Inspicio/wwwroot/js/createDropzone.js

```diff
@@ -40,7 +40,7 @@ the server when create new is pressed.
         autoProcessQueue: false,
         init: function () {
             this.on("addedfile", function (file) {
-                if (this.files[1] != null) {
+                if (this.files[1] !== null) {
```

Comment:
In JavaSciprt, you can just use `if(this.files[1])` to work out if something is [truthy](https://developer.mozilla.org/en/docs/Glossary/Truthy) (yes, I know it's a bit odd - but that's JavaScript for you)